gammapy/utils/coordinates/fov.py

```diff
@@ -1,7 +1,91 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-from astropy.coordinates import SkyCoord, SkyOffsetFrame
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import (
+    AltAz,
+    Angle,
+    BaseCoordinateFrame,
+    CoordinateAttribute,
+    DynamicMatrixTransform,
+    EarthLocationAttribute,
+    FunctionTransform,
+    RepresentationMapping,
+    SkyCoord,
+    SkyOffsetFrame,
+    TimeAttribute,
+    UnitSphericalRepresentation,
+    frame_transform_graph,
+)
+from astropy.coordinates.matrix_utilities import matrix_transpose, rotation_matrix
 
-__all__ = ["fov_to_sky", "sky_to_fov"]
+__all__ = ["FoVFrame", "fov_to_sky", "sky_to_fov"]
+
+reflect_lon_matrix = np.array([[1, 0, 0], [0, -1, 0], [0, 0, 1]])
+
+
+class FoVFrame(BaseCoordinateFrame):
+    """
```

Comment:
In the final version, it would be very nice to have  in the documentation a picture of the alignment of the different frames ;)